modules/claims_api/db/migrate/20191118152927_add_vbms_id_fields_to_claims_api_power_of_attorney.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class AddVbmsIdFieldsToClaimsApiPowerOfAttorney < ActiveRecord::Migration[5.2]
+  def change
+    add_column :claims_api_power_of_attorneys, :vbms_new_document_version_ref_id, :string
```

Comment:
do you think we should serialize any of these fields back out to the conumer or are we storing them meerly for tracking purposes?

Reply:
My thought was for debugging because you would need access to the VBMS app to do anything with them. Mainly it was so we could verify with VBMS when a file we said was uploaded was actually uploaded. However, I don't think it would hurt to send them in the status endpoint.

Reply:
yeah that makes sense, dunno how much value it would provide to the consumer

Reply:
cool cool. We can always add them down the line if something useful comes up